fairscale/experimental/nn/ssd_offload.py

```diff
@@ -358,19 +423,29 @@ def save(
 class SsdParameter(SsdTensorHandle, torch.nn.Parameter):
     @classmethod
     def from_tensor(cls: Type[SsdParameter], tensor: SsdTensorHandle) -> SsdParameter:  # type: ignore
-        r = cls(tensor.shape, tensor.dtype, tensor.requires_grad)
+        r = cls(tensor.shape, tensor.dtype, tensor.requires_grad, device=tensor.device)
```

Comment:
Why do we now need the device specification here?

Reply:
If someone tries to do something crazy like create an SsdParameter from a cuda Tensor, the ssd_param.tensor.device will be "cuda:#", but the ssd_param.device would be "cpu". It would probably when doing .to_file() but I feel like it is more accurate to have the ssd_param tensor metadata reflect the from_tensor object accurately.